charts/govuk-jobs/templates/content-store-mongo-to-postgres-cronjob.yaml

```diff
@@ -167,3 +166,103 @@ spec:
                   mountPath: /mongo-export
                 - name: app-tmp
                   mountPath: /tmp
+                - name: pg-dump
+                  mountPath: /pg-dump
+            - name: dump-postgresql
+              image: "{{ .Values.images.ContentStorePostgresqlBranch.repository }}:{{ .Values.images.ContentStorePostgresqlBranch.tag }}"
+              imagePullPolicy: "Always"
+              command: ["pg_dump"]
+              args:
+                - "-F c"
+                - "-f /pg-dump/content_store_production"
+                - "${DATABASE_URL}"
+              env:
+                - name: DATABASE_URL
+                  valueFrom:
+                    secretKeyRef:
+                      name: "{{ .Values.contentStoreMongoPostgresCron.postgresImport.databaseUrlSecretName }}"
+                      key: DATABASE_URL
+              {{- with .Values.resources }}
+              resources:
+                {{- . | toYaml | trim | nindent 16 }}
+              {{- end }}
+              securityContext:
+                allowPrivilegeEscalation: false
+                readOnlyRootFilesystem: true
+              volumeMounts:
+                - name: app-tmp
+                  mountPath: /tmp
+                - name: pg-dump
+                  mountPath: /pg-dump
+            - name: compress-pg-dump
+              image: "{{ .Values.images.ContentStorePostgresqlBranch.repository }}:{{ .Values.images.ContentStorePostgresqlBranch.tag }}"
+              imagePullPolicy: "Always"
+              command: ["tar"]
```

Comment:
I'd probably at least do the tar/gzip in a shell pipeline so as to reduce the impact on local disk.